### PR TITLE
#1691 removing unused class

### PR DESCRIPTION
**Pull request by mojaray2k**
## Description
I believe there is a lot of old SCSS for styling features that have been removed from the site, in particular the messaging-related SCSS. Example: https://github.com/department-of-veterans-affairs/vets-website/pull/10701/files/8f076cd09933799828f9f2f81be2003f559e2e5e#r322813067

The unused classes and maybe even entire .scss files should be removed if they are not used anywhere in the app.
## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by erikphansen** [comment]
@mojaray2k after standup I'll follow up with some pointers/hints on tracking down some more messaging-related styles we can remove from the codebase

**Comment by erikphansen** [comment]
@mojaray2k A little history: we used to have much more robust messaging features on the website, but they were removed as part of this massive PR: https://github.com/department-of-veterans-affairs/vets-website/commit/a09f44760ec899895da61d7d098796871f7c9e89 (you can see a ton of files in the `src/applications/messaging/` dir were deleted). That PR probably should have included deleting all of the relevant SCSS files as well. Right now the only messaging-related component we have on the site is the MessagingWidget. So the only messaging-related SCSS we need to save will be the files/classes that style that widget. We certainly don't need to remove every last trace of old, unused messaging-related SCSS, but we can get rid of a lot more than this.

**Comment by mojaray2k** [comment]
@erikphansen I removed all the unused `.scss` files that are not being used by the **MessagingWidget**

File: src/applications/personalization/dashboard/sass/messaging/messaging.scss [hunk 1]
```diff
@@ -32,15 +32,5 @@ $messaging-label-width: 10rem;
   }
 }
 
-@import "partials/message-category";
-@import "partials/message-compose";
-@import "partials/messaging-app";
-@import "partials/messaging-count-nav";
 @import "partials/messaging-folder";
```
Comment (erikphansen) [inline src/applications/personalization/dashboard/sass/messaging/messaging.scss]: these styles are being used?

File: src/applications/personalization/dashboard/sass/messaging/messaging.scss [hunk 2]
```diff
@@ -32,15 +32,5 @@ $messaging-label-width: 10rem;
...
-@import "partials/messaging-message";
-@import "partials/messaging-thread";
-@import "partials/messaging-modal";
 @import "partials/messaging-moveto";
```
Comment (erikphansen) [inline src/applications/personalization/dashboard/sass/messaging/messaging.scss]: these styles are being used?

Comment (mojaray2k) [reply src/applications/personalization/dashboard/sass/messaging/messaging.scss]: Ok they have been put back

File: src/applications/personalization/dashboard/sass/messaging/messaging.scss (hunk 1 above)
Comment (mojaray2k) [reply src/applications/personalization/dashboard/sass/messaging/messaging.scss]: Ok they have been put back

**Comment by erikphansen** [comment]
Sorry my two comments were on the lines that you _didn't_ delete. Are those two files still used at all or can we remove the imports and delete the files?

**Comment by mojaray2k** [comment]
> Sorry my two comments were on the lines that you _didn't_ delete. Are those two files still used at all or can we remove the imports and delete the files?

So the only files I saw being used where `@import "partials/messaging-folder";` and `@import "partials/messaging-moveto";` but I put back the ones you said are being used even though I could not find where they were being used.

**Comment by erikphansen** [comment]
@mojaray2k Cool. If `partials/messaging-folder` and `partials/messaging-moveto` are still being used, leave them. I just didn't think they were so wanted to confirm that they shouldn't be removed. But I never requested that you restore the imports you removed in your original commit. Please go ahead and delete those imports and the files themselves.

**Comment by mojaray2k** [comment]
> Sorry my two comments were on the lines that you _didn't_ delete. Are those two files still used at all or can we remove the imports and delete the files?
Those are the only two files I see in use @erikphansen

**Comment by erikphansen** [comment]
@mojaray2k from my above note:

>  I never requested that you restore the imports you removed in your original commit. Please go ahead and delete those imports and the files themselves.

Thanks.

**Comment by mojaray2k** [comment]
> @mojaray2k from my above note:
> 
> > I never requested that you restore the imports you removed in your original commit. Please go ahead and delete those imports and the files themselves.
> 
> Thanks.

@erikphansen These files and imports have been removed.

**Review by erikphansen — APPROVED** [review]

**Comment by erikphansen** [comment]
@department-of-veterans-affairs/frontend-review-group this is ready for your review now!

**Review by rianfowler — APPROVED** [review]

**Pull request merged**